mesonbuild/backend/ninjabackend.py

```diff
@@ -151,6 +151,7 @@ def __init__(self, build):
         self.fortran_deps = {}
         self.all_outputs = {}
         self.introspection_data = {}
+        self.created_llvm_ir_rule = PerMachine(False, False, False)
```

Comment:
Field was being added dynamically below. This is cleaner and easier on both the code the type checker.